CONTRIBUTING.md

```diff
@@ -24,9 +24,9 @@ If you want to get in touch with us first before contributing, you can use:
 
 [issues]: https://github.com/strimzi/strimzi-kafka-operator/issues
 [newbie-issues]: https://github.com/strimzi/strimzi-kafka-operator/labels/good-start
-[hacking-guide]: https://github.com/strimzi/strimzi-kafka-operator/blob/master/HACKING.md
+[hacking-guide]: https://github.com/strimzi/strimzi-kafka-operator/blob/master/development-docs/DEV_GUIDE.md
 [dev-quick-start]: https://github.com/strimzi/strimzi-kafka-operator/blob/master/DEV_QUICK_START.md
```

Comment:
this path is now wrong too.

I'd argue that maybe we should give deep links as opposed to hyperlinks?